packages/cli/src/databases/entities/index.ts

```diff
@@ -70,4 +73,7 @@ export const entities = {
 	TestCaseExecution,
 	Folder,
 	FolderTagMapping,
+	InsightsRaw,
```

Comment:
instead of adding all these here, maybe we can have the modules define the entities specific to them? then we'd have to generate a new array of all the entity classes before the `Db.init` call, but we can then move all the entities and repositories in the module as well.

Reply:
I asked Tomi about this and he said to keep the db stuff outside of the module. Personally I don't have a strong opinion, the db layout does not change whether we load the entities or not, so no harm in loading them lazily. 

Give me a shout and I'll make a PR extracting this.